libs/public-pages/src/pages/summary-of-publications/en.ts

```diff
@@ -0,0 +1,7 @@
+export const en = {
+  titlePrefix: "What do you want to view from",
+  noPublicationsMessage: "Sorry, no lists found for this court",
+  backLink: "Back",
```

Comment:
Is backlink used anywhere in the template? govukBacklink is include in base-template.njk in webcore. The text for back link is actually in apps/web/locales

Reply:
Removed